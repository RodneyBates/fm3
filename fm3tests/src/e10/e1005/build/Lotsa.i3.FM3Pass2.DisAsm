 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L2(16_2),[1,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L1(16_1),[3,2])
 ItkTypeDeclLt([3,7])
 ItkDeclNo(D1,[3,7])
 ItkTypeDeclType([3,10])
 ItkDefTopExprNo(L1(16_1),[3,12])
 ItkTypeDeclRt([3,7])
 ItkOpenDeclListRt(L1(16_1),[3,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkInterfaceRt(L2(16_2),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
