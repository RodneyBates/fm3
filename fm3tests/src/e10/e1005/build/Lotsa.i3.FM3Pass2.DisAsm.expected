 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L2(16_2),[1,0])
 ItkBlockDeclListLt(L1(16_1),[3,2])
 ItkTypeDeclLt([3,7])
 ItkDeclNo(D1,[3,7])
 ItkTypeDeclEquals([3,10])
 ItkReservedId(Rid"CHAR",[3,12])
 ItkTypeDeclRt([3,7])
 ItkBlockDeclListRt(L1(16_1),[3,2])
 ItkInterfaceRt(L2(16_2),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
